src/_community/pluralkit.md

```diff
@@ -1,62 +1,53 @@
 ---
-title: Tupperbox
+title: PluralKit
+redirect_from:
+  - /community/tupperbox
 ---
 
 {% admonition %}
-**Note:** This document is specific to Discord. If you're not making use of our community spaces on Discord, Tupperbox
+**Note:** This document is specific to Discord. If you're not making use of our community spaces on Discord, PluralKit
 won't be available.
 {% endadmonition %}
 
-[Tupperbox](https://github.com/Runi-c/Tupperbox) is a Discord bot that exists to allow users to proxy their messages
-via Discord webhooks. This has many practical applications, especially given that Discord doesn't have any features
-that support the plural community, or any built-in mental health aids.
+[PluralKit](https://pluralkit.me) is a Discord bot that exists to allow users to proxy their messages via Discord 
+webhooks. This has many practical applications, especially given that Discord doesn't have any features that support 
+the plural community, or any built-in mental health aids.
 
-Tupperbox has many fun and interesting uses for all kinds of situations, but in Quilt community spaces it should only 
-be used for [plurality](#plurality) or self-identity purposes, or as a mental health aid. We do not allow users to make use of 
-Tupperbox for role-playing.
+PluralKit has all kinds of fun and interesting uses, but in Quilt community spaces it should only be used for 
+[plurality](#plurality) or self-identity purposes, or as a mental health aid. We do not allow users to make use of 
```

Comment:
Should be "plurality, self-identity purposes, or as a mental health aid"

Reply:
While technically correct, I think the text as written here flows a lot better haha

Reply:
Fair enough I suppose

(also thanks github for not sending me a notification of this)